### PR TITLE
Change command shell script comment

### PR DESCRIPTION
**Pull request by sarojkmb**
- I did not change anyty comand 
- I changed on some coment lines

### CODE REVIEW

**Comment by sarojkmb** [comment]
@dibyashasahu : please review

File: linux-commands/command-1.sh [hunk 1]
```diff
@@ -50,6 +50,9 @@ FILES_DIR="/Users/saroj/Workspace/projects/all-programming-language/linux-comman
 
 # com command
 
+# this is my line on develop
+# I want to merge this in master
```
Comment (dibyashasahu) [inline linux-commands/command-1.sh:54]: @saroj here is the mistake

Comment (sarojkmb) [reply linux-commands/command-1.sh:54]: ok, i will change

File: linux-commands/command-1.sh [hunk 2]
```diff
@@ -50,6 +50,9 @@ FILES_DIR="/Users/saroj/Workspace/projects/all-programming-language/linux-comman
 
 # com command
 
+# this is my line on develop
```
Comment (dibyashasahu) [inline linux-commands/command-1.sh:53]: hi

File: linux-commands/command-1.sh [hunk 3]
```diff
@@ -50,6 +50,9 @@ FILES_DIR="/Users/saroj/Workspace/projects/all-programming-language/linux-comman
...
+# I want to merge this in master
+# ok changed
```
Comment (dibyashasahu) [inline linux-commands/command-1.sh:55]: bye

Comment (sarojkmb) [reply linux-commands/command-1.sh:55]: ok changed done

**Pull request merged**